examples/mbedTLS/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-mbedtls)
+
+hunter_add_package(mbedtls)
+find_library(
```

Comment:
Thanks for those examples.

>There are a lot of boilerplate code but it's not very hard actually.

What I meant by `might take some time` was that we might have to wait for the mbedtls and aws-iot-device-sdk-cpp maintainers to actually merge my PR regarding CMake config code generation in their respective projects.

Anyways, I will give it a shot.

It would be nice if all maintainers are as active as you are :)